app/scripts/translations.js

```diff
@@ -1,5 +1,11 @@
 angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
 /* jshint -W100 */
 	gettextCatalog.setStrings('de', {"My Profile":"DE - My Profile"});
+    gettextCatalog.setStrings('en', {
+        "URGENCY": "News Value",
+        "Urgency": "News Value",
+        "Urgency stats": "News Value stats",
+        "urgency": "News Value"
```

Comment:
Why we have so many (3) keys (with just different size of letters) keys for this same translation string? Translators will need translate  "News Value" at least 3 times that string.